apps/src/templates/codeDocs/MethodSummaryTable.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import EnhancedSafeMarkdown from '@cdo/apps/templates/EnhancedSafeMarkdown';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import color from '@cdo/apps/util/color';
+import {tableLayoutStyles} from '@cdo/apps/templates/tables/tableConstants';
+import {TextLink} from '@dsco_/link';
+
+export default function MethodSummaryTable({
+  methods,
+  programmingClassLink,
+  openLinkInNewTab = false
+}) {
+  return (
+    <table style={{...tableLayoutStyles.table, width: '100%'}}>
+      <tbody>
+        {methods.map(method => (
+          <tr key={method.key}>
+            <td style={styles.method}>
+              <h3>
+                {method.name}
+                {programmingClassLink && (
```

Comment:
I'll leave the link decision up to @moneppo but would it make sense to just make the entire method name a link instead?

Reply:
I'm open to either! Unless I hear otherwise, I'll keep as-is for now and merge this when staging reopens so I don't forget about it.